src/main/java/catan/controllers/PlayController.java

```diff
@@ -269,11 +271,13 @@ public void acceptTradeProposition(@RequestParam(value = "token", required = fal
             method = POST,
             produces = APPLICATION_JSON_VALUE)
     public void declineTradeProposition(@RequestParam(value = "token", required = false) String token,
-                                        @RequestParam(value = "gameId", required = true) String gameId) throws AuthenticationException, GameException, PlayException {
+                                        @RequestParam("gameId") String gameId,
+                                        @RequestParam("offerId") String offerId) throws AuthenticationException, GameException, PlayException {
```

Comment:
gameId and offerId should be mandatory parameters, you should add ", required = true" to RequestParam definition